src/terrain/routes/filesystem.clj

```diff
@@ -27,7 +27,10 @@
 (defn- fix-sort-col-param [param-value]
   (as-> param-value v
     (string/lower-case v)
-    (if (= v "lastmodified") "datemodified" v)))
+    (cond
+      (= v "id")           "path"
```

Comment:
I'm not aware of a time when the ui sends an `id` sort column, but I could definitely be missing something. I guess even if not, then this change shouldn't hurt anything.

Reply:
Ah, thanks for checking on that. The old version of this endpoint that translated the sort column from `id` to `path`, so I thought I should reinsert it here in case it was used anywhere. I think I might leave it for now. We can always remove it later if we need to.

Reply:
The UI used to treat `id` the same as `path` before we added UUIDs to file/folder metadata, so this was probably some cruft from those days; but it's also probably fine leaving this check until we retire the old UI.